<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PMagnet13</class>
 <widget class="QWidget" name="PMagnet13">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>232</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="img_magnet">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>550</width>
       <height>350</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../Resources/pyleecan.qrc">:/images/images/MachineSetup/P_Magnet/Surface Magnet type 13.PNG</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="in_Wmag">
         <property name="text">
          <string>Wmag :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="FloatEdit" name="lf_Wmag">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="unit_Wmag">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="in_Hmag">
         <property name="text">
          <string>Hmag  :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="FloatEdit" name="lf_Hmag">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="unit_Hmag">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="in_Rtopm">
         <property name="text">
          <string>Rtopm  :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="FloatEdit" name="lf_Rtopm">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="unit_Rtopm">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="in_H0">
         <property name="text">
          <string>H0      :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="FloatEdit" name="lf_H0">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="unit_H0">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="WMagnetOut" name="w_out" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FloatEdit</class>
   <extends>QLineEdit</extends>
   <header>pyleecan.GUI/Tools/FloatEdit.h</header>
  </customwidget>
  <customwidget>
   <class>WMagnetOut</class>
   <extends>QWidget</extends>
   <header>pyleecan.GUI.Dialog.DMachineSetup.SMagnet.WMagnetOut.WMagnetOut.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lf_Wmag</tabstop>
  <tabstop>lf_Hmag</tabstop>
  <tabstop>lf_Rtopm</tabstop>
  <tabstop>lf_H0</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../Resources/pyleecan.qrc"/>
  <include location="../../../../Resources/pyleecan.qrc"/>
  <include location="../../../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
